test/integration_test.go

```diff
@@ -421,7 +421,12 @@ func (ts *IntegrationTestSuite) TestChildWFWithParentClosePolicyTerminate() {
 	ts.NoError(err)
 	resp, err := ts.libClient.DescribeWorkflowExecution(context.Background(), childWorkflowID, "")
 	ts.NoError(err)
+	// Need to wait for child workflow to finish as well otherwise test becomes flaky
+	err = ts.waitForWorkflowFinish(resp.WorkflowExecutionInfo.Execution.GetWorkflowId(), resp.WorkflowExecutionInfo.Execution.GetRunId())
```

Comment:
this error is not checked

Reply:
Not checked by design. since this call can throw terminated error which we do not care about.